setup.py

```diff
@@ -15,24 +15,27 @@
 # limitations under the License.
 # ===============================================================================
 
+import distutils.command.build as orig_build
+import glob
+
 # System imports
 import os
+import pathlib
 import sys
 import time
-from setuptools import setup, Extension
-from setuptools.command.build_ext import build_ext as _build_ext
-import setuptools.command.develop as orig_develop
-import distutils.command.build as orig_build
-from os.path import join as jp
-import pathlib
+from concurrent.futures import ThreadPoolExecutor
 from distutils.sysconfig import get_config_vars
-from Cython.Build import cythonize
-import glob
+from os.path import join as jp
+
 import numpy as np
-from scripts.version import get_onedal_version
+import setuptools.command.develop as orig_develop
+from Cython.Build import cythonize
+from setuptools import Extension, setup
+from setuptools.command.build_ext import build_ext as _build_ext
+
 import scripts.build_backend as build_backend
 from scripts.package_helpers import get_packages_with_tests
-from concurrent.futures import ThreadPoolExecutor
+from scripts.version import get_onedal_version
```

Comment:
Sorry I have reviewed this part before, just forget submit review commit. Now it looks outdated

Reply:
This is exactly what `isort` is doing and where the changes came from. Sometimes (when paths are not configured properly) it mixes third-party and local imports. Did you see that happening here? In this case I can update the config. But not in this PR, since I removed the changes in `setup.py` anyway.